eox_nelp/mfe_config_api/urls.py

```diff
@@ -0,0 +1,10 @@
+"""eox_nelp mfe_config_api  urls
+"""
+from django.conf.urls import url, include
+
+
+app_name = 'eox_nelp'  # pylint: disable=invalid-name
+
+urlpatterns = [  # pylint: disable=invalid-name
+    url(r'^v1/', include('eox_nelp.mfe_config_api.v1.urls', namespace='mfe-config-api-v1')),
```

Comment:
I think it is not necessary a regex and since we are refactoring this code we could implement path instead of url

Reply:
[054f5d0](https://github.com/eduNEXT/eox-nelp/pull/23/commits/054f5d08d7eb1ed24ce021b7d4e24e9d628173b9)